### PR TITLE
Add a section on creating a run config.

### LINKED ISSUE #322: Add more detail on how to create a new Liberty run/debug configuration
Either update the [Debug your application](https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/docs/user-guide.md#debug-your-application) section or add a new section to the user guide.

### PR DESCRIPTION
**Pull request by turkeylurkey**
A preview:
image

Fixes #322

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -159,6 +160,13 @@ After the server starts, the IntelliJ IDEA debugger attaches and switches to the
 
 ![Liberty debugger attached](images/liberty-debugger-attached.png)
 
+## Create a new Liberty Run/Debug Configuration
+
+The Liberty Tools plugin contributes a new type of Run/Debug configuration. Select the **Run -> Edit Configurations** menu to open the **Edit Run/Debug configurations** dialog. Click **+** and select Liberty from the menu. 
+
+![Liberty new configuration](images/liberty-new-config.png)
+
+For more information see the [IDE documentation](https://www.jetbrains.com/help/idea/run-debug-configuration.html).
```
Comment (dmuelle) [inline docs/user-guide.md:169]:
```suggestion
For more information, see [Run/debug configurations](https://www.jetbrains.com/help/idea/run-debug-configuration.html) in the Intellij IDEA documentation.
```

File: docs/user-guide.md [hunk 2]
```diff
@@ -159,6 +160,13 @@ After the server starts, the IntelliJ IDEA debugger attaches and switches to the
 
 ![Liberty debugger attached](images/liberty-debugger-attached.png)
 
+## Create a new Liberty Run/Debug Configuration
+
+The Liberty Tools plugin contributes a new type of Run/Debug configuration. Select the **Run -> Edit Configurations** menu to open the **Edit Run/Debug configurations** dialog. Click **+** and select Liberty from the menu.
```
Comment (dmuelle) [inline docs/user-guide.md:165]:
```suggestion
The Liberty Tools plugin contributes a its own Run/Debug configuration. Select the **Run -> Edit Configurations** menu to open the **Edit Run/Debug configurations** dialog. Click **+** and select Liberty from the menu. 
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
A few minor suggestions- also, we've explained how to create this debug config, but haven't said anything about why you might want to use it- if it is specific to LT, are there Liberty-specific benefits that we should call out? What is the value add?

File: docs/user-guide.md [hunk 3]
```diff
@@ -159,6 +160,13 @@ After the server starts, the IntelliJ IDEA debugger attaches and switches to the
 
 ![Liberty debugger attached](images/liberty-debugger-attached.png)
 
+## Create a new Liberty Run/Debug Configuration
```
[the remark this answers is not in the document]
Comment (turkeylurkey) [reply docs/user-guide.md:163]: Ok

File: docs/user-guide.md (hunk 1 above)
Comment (turkeylurkey) [reply docs/user-guide.md:169]: Also ok.

File: docs/user-guide.md [hunk 4]
```diff
@@ -19,6 +19,7 @@ This guide provides detailed instructions on how to import and configure your Li
    - [Gradle-built applications](#gradle-built-applications)
 - [Stop your application](#stop-your-application)
 - [Debug your application](#debug-your-application)
+- [Create a new Liberty Run/Debug Configuration](#create-a-new-liberty-rundebug-configuration)
```
Comment (dmuelle) [inline docs/user-guide.md:22]:
```suggestion
- [Create a Liberty Run/Debug Configuration](#create-a-liberty-rundebug-configuration)
```

Comment (turkeylurkey) [reply docs/user-guide.md:22]: Yes.

**Comment by turkeylurkey** [comment]
I added a sentence to point out the use case for creating a configuration.

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Pull request merged**